builtin/providers/openstack/resource_openstack_lb_loadbalancer_v2.go

```diff
@@ -57,6 +57,12 @@ func resourceLoadBalancerV2() *schema.Resource {
 				ForceNew: true,
 			},
 
+			"vip_port_id": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
```

Comment:
IF this is exported only, then we shouldn't have the `Optional: true` - this allows the user to set it. But we never use the value